packages/chevrotain/rollup.config.js

```diff
@@ -0,0 +1,82 @@
+const typescriptPlugin = require("@rollup/plugin-typescript")
+const { default: nodeResolvePlugin } = require("@rollup/plugin-node-resolve")
+const typescript = require("typescript")
+const { terser } = require("rollup-plugin-terser")
+const commonjsPlugin = require("@rollup/plugin-commonjs")
+
+const pkg = require("./package.json")
+
+module.exports = [
+  // no deps
+  {
+    input: "src/api.ts",
+    output: [
+      ["commonjs", false],
+      ["es", false]
+    ].map(([format, compress]) => ({
+      format: format,
+      entryFileNames:
+        "chevrotain" +
+        (compress ? ".min" : "") +
+        (format === "es" ? ".mjs" : ".js"),
+      sourcemap: true,
+      dir: "lib",
+      name: "chevrotain",
+      exports: "named"
+    })),
+    external: Object.keys(pkg.dependencies),
+    plugins: [typescriptPlugin({ typescript })],
+    onwarn: function (warning, warn) {
+      if ("THIS_IS_UNDEFINED" === warning.code) return
+      // if ('CIRCULAR_DEPENDENCY' === warning.code) return
+      warn(warning)
+    }
+  },
+
+  // bundle
+  {
+    input: "src/api.ts",
+    output: [
+      ["es", false, true],
+      ["es", true, true],
+      ["umd", false, true],
+      ["umd", true, true]
+    ].map(([format, compress]) => ({
+      format: format,
+      entryFileNames:
+        "chevrotain-bundle" +
+        (compress ? ".min" : "") +
+        (format === "es" ? ".mjs" : ".js"),
+      sourcemap: true,
+      dir: "lib",
+      name: "chevrotain",
+      exports: "named",
+      plugins: compress
+        ? [
+            terser({
+              compress: {
+                passes: 3,
+                unsafe: true,
```

Comment:
How do we know everything works after bundling particularly when using `unsafe` minification modes? what benefit does `unsafe` produce in the bundle size?